src/snappy_device_agents/devices/muxpi/muxpi.py

```diff
@@ -244,11 +245,20 @@ def check_path(dir):
             # Not a limerick image
             pass
 
+        try:
+            disk_info_path = (
+                self.mount_point / "writable/lib/firmware/*-tegra/"
+            )
+            self._run_control(f"ls {disk_info_path} &>/dev/null")
+            return "tegra"
+        except ProvisioningError:
+            # Not a tegra image
+            pass
+
```

Comment:
I think this should work, but couldn't you just add this to IMAGE_PATH_IDS at the top of the class and have it check the same way we check for other image types already?

Reply:
Thank you, actually that's exactly what I tried in the first time, but the "test -e" command isn't compatible with a wildcard path... And I couldn't find a better way to catch it... So I ended up adding this other way, mimicking the limerick case

Reply:
Weird, I tried it locally and it seems to work with wildcards on my system. I wonder what the difference is?
I'm fine with this though.